healthcheck/Dockerfile

```diff
@@ -1,13 +1,10 @@
 FROM bitnami/minideb:stretch
 
 RUN apt-get update && apt-get -y install git
-RUN git clone https://github.com/BayLibre/lava-healthchecks-binary.git
+RUN git clone https://github.com/kernelci/lava-healthchecks-binary.git lava-healthchecks-binary
```

Comment:
This repository do not exists. And lava-docker will migrate off kernelci to baylibre soon.

Reply:
Oh I thought it was going to gitlab.com under the Linaro / LAVA org?

Reply:
Since lava-docker is not a Linaro project, it is easier to go to baylibre. It will requires less "spetial rigths".